databuilder/databuilder/extractor/query/query_metadata_base.py

```diff
@@ -0,0 +1,354 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from abc import abstractmethod
+from typing import (
+    Any, Dict, Iterator, List, Optional, Union,
+)
+
+from pyhocon import ConfigTree
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.models.query import (
+    QueryExecutionsMetadata, QueryJoinMetadata, QueryMetadata, QueryWhereMetadata,
+)
+from databuilder.models.table_metadata import ColumnMetadata, TableMetadata
+from databuilder.models.user import User as UserMetadata
+
+LOGGER = logging.getLogger(__name__)
+
+
+class QueryMetadataBaseExtractor(Extractor):
+    """
+    Provides common funcitons used by query metadata extractors including:
+        - Converting a list of Stemma SQL tables to QueryMetadata,
```

Comment:
please remove company mentioned in comment.

Reply:
Also could you give an example of SQL table in your use case ? cc @markgrover

Reply:
The SQL Table referenced here is simply a Python object that is used to serialize a "table" from our proprietary SQL parsing into an Amundsen format. To your point below, all references to this should have been removed.